.planning/milestones/v1.1-phases/01-pushdispatcher-refactor-no-behaviour-change/01-01-PLAN.md

```
@@ -0,0 +1,1135 @@
+! grep -qE "info!|warn!|error!|debug!|trace!" src/push/dispatcher.rs
+
+# Must-not #11: no name() method added to PushService trait
+! grep -qE "fn name\(&self\)" src/push/mod.rs src/push/fcm.rs src/push/unifiedpush.rs
+```
+</verification>
+
+<success_criteria>
+The plan is complete when ALL of the following are true:
+
+1. The 6 files in `files_modified` are the **only** files modified by this commit. `git diff --name-only HEAD~1` returns exactly:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)